pkg/deviceplugin/lister.go

```diff
@@ -16,10 +16,10 @@ const (
 )
 
 type macvtapConfig struct {
-	Name     string `json:"name"`
-	Master   string `json:"master"`
-	Mode     string `json:"mode"`
-	Capacity int    `json:"capacity"`
+	Name        string `json:"name"`
+	LowerDevice string `json:"lowerDevice"`
```

Comment:
The key does not match your documentation changes where it was all lower case

Reply:
Done.